src/app/components/ThemeProvider/spacings.ts

```diff
@@ -0,0 +1,6 @@
+import pixelsToRem from '../../utilities/pixelsToRem';
+
+export const FULL = pixelsToRem(16);
```

Comment:
I think our current implementation considers "full" to be half a REM

Our current version of these includes the `rem` - eg `'1rem'` rather than `1`

I think this is better in both aspects - just wanted to make sure it was intentional

Reply:
oh, I thought a GEL spacing was 1rem/16px. I'll fix this.

Yeh I think it's better to stick with numbers so we can more easily operate on the values e.g. `{spacings.FULL / 4}rem` for a quarter spacing.